.circleci/config.yml

```diff
@@ -161,48 +161,56 @@ workflows:
           image_tag: heroku/pack:20
           image_file: pack-20-run.tar
       - create-pack-builder:
-          name: create-service-builder
+          name: create-service-builder-18
           image_tag: heroku/buildpacks:18
           image_file: buildpacks-18.tar
-          builder_toml: builder.toml
+          builder_toml: builder-18.toml
           requires:
             - create-18-run-image
             - create-18-build-image
+      - create-pack-builder:
```

Comment:
This image is only being generated, not published. If this isn't intentional (eg to allow CI to run until remaining CNB support added), we'll need to also:
1. rename the `publish-service-builder` task to `publish-service-builder-18`
2. add a new `publish-service-builder-20` task
3. set up the getting started guide tests for 20 too